examples/v0/account_management/get_accessible_customers.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example lists the resource names for the customers that the
+authenticating user has access to.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+_DEFAULT_LIMIT=200
```

Comment:
Why is this limit required? Google Ads limit this to 20 accounts anyway.

https://support.google.com/google-ads/answer/6139186?hl=en

Reply:
It was a suggestion from @wihl and @jradcliff for the larger clients that may have thousands of customers. It just restricts the number of results that are logged, it doesn't affect the actual request. Also those docs say that up to 20 customers can be associated with a single email address, but does that also apply to customers that are connected to a manager account? I thought those were different concepts. 

I was curious if the API itself would restrict the number of results returned here, but I couldn't find any documentation to support that. Also there doesn't appear to be a way to paginate these results. We figured a limit of 200 was a reasonable number to keep logs manageable.

Reply:
@AnashOommen raises an excellent point that I failed to recognize: the request will only return customers that are *directly* accessible by the user, and that is indeed limited to 20 customers. +1 to removing this parameter and limit, and sorry I didn't think of this earlier.

Reply:
@jradcliff @AnashOommen No problem! I'll strip it out now.